app/src/main/res/raw/errorpage_style.css

```diff
@@ -0,0 +1,229 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  height: 100%;
+  --moz-vertical-spacing: 10px;
```

Comment:
(IIUC) These are actually css variables - AFAICS prefixes have a single dash (-moz-something), and two dashes (--something) are variables. The use of --moz is confusing, but it seems easiest to not modify what the original page is using?